codemods/utils/mappings.ts

```diff
@@ -132,6 +132,8 @@ const supportedPropertyValuePairs = [
 ];
 
 const supportedProperties = [
+  /^flexGrow/,
+  /^flexShrink/,
```

Comment:
i'm adding these for posterity, but they actually have been making zero difference in this.